airbyte-integrations/connectors/source-google-webfonts/integration_tests/sample_config.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
can you update the sample config? Since the config.json under secrets is not included, the sample file would be useful for the maintainer to create one. Thanks.

Reply:
Done